Arduino/McLighting/request_handlers.h

```diff
@@ -765,7 +821,7 @@ void webSocketEvent(uint8_t num, WStype_t type, uint8_t * payload, size_t lenght
     }
 
     void sendState() {
-      const size_t bufferSize = JSON_OBJECT_SIZE(3) + JSON_OBJECT_SIZE(6);
+      const size_t bufferSize = JSON_OBJECT_SIZE(3) + JSON_OBJECT_SIZE(6) + 500;
```

Comment:
Latest Arduino JSON version is very aggressive with memory managgement, it won't send any data just because of buffer length being small. The previous code without those changes would not work correctly.